seedlet-blocks/sass/theme.scss

```diff
@@ -1,6 +1,8 @@
 @import './blocks/site-title.scss';
 @import './blocks/links.scss';
 @import './blocks/latest-posts';
+@import './blocks/post-comments';
```

Comment:
I don't know why there are seedlet blocks changes in here, maybe I branched off of something I didn't mean to.